lib/internal/errors.js

```diff
@@ -86,3 +86,4 @@ module.exports = exports = {
 // Note: Please try to keep these in alphabetical order
 E('ERR_ASSERTION', (msg) => msg);
 // Add new errors from here...
+E('ERR_UNK_ENCODING', 'Unknown encoding: %s');
```

Comment:
I think I used the `UNK_` in one of mine... let's go ahead and expand that out to `UNKNOWN_`